lua/navi/api.lua

```diff
@@ -34,8 +34,8 @@ function M.request_with_context(cfg, buf, from_row, to_row)
 
     log.d(vim.inspect({
         buf = buf,
-        start_position = start_position,
-        end_position = end_position,
+        -- start_position = start_position,
```

Comment:
Ville du ikke ha denne logginga?

Reply:
Var lsp serveren som mente at disse to verdiene ikke fantes, skal ta en ny titt